src/main/java/pinger/Pinger.java

```diff
@@ -26,6 +26,8 @@ public static void main(String[] args) throws IOException {
                 addField("Header Fields", conn.getHeaderFields());
                 addField("Content", content);
             }
+        } catch(Exception e) {
+            System.out.println(e);
```

Comment:
You can call `e.printStackTrace()` instead to print out the stack trace. Also print out a message saying that an error occured.